docker/Makefile

```diff
@@ -10,7 +10,7 @@ TAG := ${REPO}:$(shell date -u +%Y%m%dT%H%M%SZ)
 tag: $(FILES)
 	docker build -t ${TAG} .
 	echo ${TAG} > tag
-	docker tag -f ${TAG} ${REPO}
+	docker tag ${TAG} ${REPO}
```

Comment:
why -f is not needed

Reply:
The flag -f is not supported anymore by docker.
It seems to be doing the tagging just fine even without the -f flag